packages/ui/src/composables/useValidation.ts

```diff
@@ -42,7 +42,7 @@ export const useValidation = (
   emit: (event: any) => any,
   reset: () => any,
   focus: () => any,
-  blur: () => any,
+  blur?: () => any,
```

Comment:
What is blur doing in this method at all?
I don't see it in `FormChild` declaration.

Reply:
It is my mistake. This can be safely removed after fixes in the components `VaInput`, `VaSelect`, `VaTimeInput`